src/Bicep.Core/TypeSystem/ResourceScope.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Bicep.Core.TypeSystem
+{
+    [Flags]
+    public enum ResourceScope
```

Comment:
Should we follow the convention to pluralize the name since it has `Flags` attribute?

Reply:
Personally I like either `ResourceScope.XYZ` or `ResourceScopeFlags.XYZ` for convention - `ResourceScopes.ResourceGroup` feels odd.

Reply:
Sounds good to me.

Reply:
Got it. Sounds good to me as well.